packages/app/src/domain/international/infected-table-tile/components/wide-infected-table.tsx

```diff
@@ -0,0 +1,184 @@
+  const highestAverage = maxBy(data, (x) => x.infected_per_100k_average);
+
+  return (
+    <Box overflow="auto" mb={3}>
+      <StyledTable>
+        <thead
+          css={css({
+            borderBottom: '1px solid lightGray',
+          })}
+        >
+          <tr>
+            <HeaderCell
+              css={css({
+                pl: 3,
+              })}
+            >
+              {text.header_land}
+            </HeaderCell>
+            <HeaderCell
+              css={css({
+                width: asResponsiveArray({
+                  sm: 150,
+                  lg: 150,
+                }),
+              })}
+            >
+              {text.header_per_inwoners}
+            </HeaderCell>
+            <HeaderCell
+              css={css({
+                textAlign: 'right',
+                pr: 3,
+                width: asResponsiveArray({
+                  sm: 150,
+                  lg: 220,
+                }),
+              })}
+            >
+              {text.header_totale}
+            </HeaderCell>
+          </tr>
+        </thead>
+
+        <tbody>
+          {data.map((item, index) => (
+            <>
+              {inputValue.length === 0 ? (
+                <>
+                  {isExpanded || index < MAX_COUNTRIES_START ? (
+                    <TableRow
+                      item={item}
+                      highestAverage={highestAverage?.infected_per_100k_average}
+                      countryNames={countryNames}
+                    />
+                  ) : null}
+                </>
+              ) : (
+                <>
+                  {matchingCountries.some(
+                    (i) => i.country_code === item.country_code
+                  ) && (
+                    <TableRow
+                      item={item}
+                      highestAverage={highestAverage?.infected_per_100k_average}
+                      countryNames={countryNames}
+                    />
+                  )}
+                </>
+              )}
+            </>
+          ))}
+        </tbody>
+      </StyledTable>
+    </Box>
+  );
+}
+
+interface tableRowProps {
+  item: InTestedOverall;
+  highestAverage: number | undefined;
+  countryNames: Record<string, string>;
+}
+
+function TableRow({ item, highestAverage, countryNames }: tableRowProps) {
+  const { formatNumber } = useIntl();
+
+  const filterBelow = getFilteredThresholdValues(
+    internationalThresholds.infected_per_100k_average,
+    item.infected_per_100k_average
+  );
+
+  return (
+    <tr
+      css={css({
+        backgroundColor:
+          item.country_code === 'NLD' ? colors.tileGray : undefined,
+      })}
+    >
+      <Cell
+        css={css({
+          pl: 3,
+        })}
+      >
+        <InlineText
+          fontWeight={item.country_code === 'NLD' ? 'bold' : undefined}
+        >
+          {countryNames[item.country_code.toLocaleLowerCase()]}
+        </InlineText>
+      </Cell>
+      <Cell>
+        {highestAverage && (
```

Comment:
when `highestAverage` is `0` a `0` will be rendered
```suggestion
        {highestAverage > 0 && (
```